guides/client/form-bindings.md

```diff
@@ -408,3 +408,44 @@ document.getElementById("my-form").dispatchEvent(
   new Event("submit", {bubbles: true, cancelable: true})
 )
 ```
+
+## Preventing form submission with JavaScript
+
+In some cases, you may want to conditionally prevent form submission based on client-side validation or other business logic before allowing a `phx-submit` to be processed by the server.
+
+The standard DOM API can be used to prevent the default form submission behavior:
+
+```javascript
+document.getElementById("my-form").addEventListener("submit", (event) => {
+  event.stopPropagation()
+  event.preventDefault()
+})
+```
+
+You can also implement this as a JavaScript hook added to the form with `phx-hook`:
+
+```javascript
+let Hooks = {}
```

Comment:
I would not mention the first case, since it is unlikely that the form will always be there when the page renders.

```suggestion
JavaScript can be used to prevent the default form submission behavior, for example with a [hook](js-interop.md#client-hooks-via-phx-hook):

```javascript
/**
 * @type {import("phoenix_live_view").HooksOptions}
 */
let Hooks = {}
```

Reply:
Yup, makes sense. I was going off of the previous header which has the same case (for dispatching events on the form - Do you want me to drop that case from the guide here or leave it for a later review?

Reply:
Let's keep the other part as is for now. By itself those snippets are fine as we're not saying where the code should be. I guess it would be fine here as well, but in a real app you'd most likely have it in a hook.

Reply:
I removed it from the new section for now - I think it's a good idea to try and orient the guides around using hooks as a best practice, that sounds good to me 👍